eslint.config.js

```diff
@@ -29,6 +29,7 @@ export default tseslint.config(
         { allowConstantExport: true },
       ],
       "no-console": "error",
+      "react-hooks/exhaustive-deps": "error",
```

Comment:
Im surprised this one is just a warning by default. So many bugs avoided by having it in my experience.